Controllers/CHumidityController.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * CHumidityController.h
+ *
+ *  Created on: Aug. 11, 2022
+ *      Author: samk
+ */
+
+#ifndef CHUMIDITYCONTROLLER_H_
+#define CHUMIDITYCONTROLLER_H_
+
+#include "CBME280.h"
+#include "CController.h"
+#include "CHumidifier.h"
+#include "CPIDLoop.h"
+#include "IHardwareMap.h"
+
+#define CHANNEL_NUMBER 1
```

Comment:
If the scope of this constant is just this class, I think a private member variable would be much better. 
`static constexpr uint8_t CHANNEL_NUMBER = 1`

Reply:
I agree